yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -410,16 +410,6 @@ export function getLogger() {
   return createDebugLogger('aztec:' + describeBlockName);
 }
 
-// docs:start:delay
-/**
- * Sleep for a given number of milliseconds.
- * @param ms - the number of milliseconds to sleep for
- */
-export function delay(ms: number): Promise<void> {
```

Comment:
This is no longer used anywhere and it's redundant to sleep function from foundation package so I nuked it.